src/app/rundown/components/rundown-header/rundown-header.component.ts

```diff
@@ -0,0 +1,89 @@
+import {Component, Input, OnChanges, OnDestroy, OnInit, SimpleChange, SimpleChanges} from '@angular/core';
+import {Rundown} from "../../../core/models/rundown";
+import {Piece} from "../../../core/models/piece";
+import {ShowStyleVariantStateService} from "../../../core/services/show-style-variant-state.service";
+import {ShowStyleVariant} from "../../../core/models/show-style-variant";
+import {SubscriptionLike} from "rxjs";
+
+@Component({
+  selector: 'sofie-rundown-header',
+  templateUrl: './rundown-header.component.html',
+  styleUrls: ['./rundown-header.component.scss']
+})
+export class RundownHeaderComponent implements OnInit, OnDestroy, OnChanges {
+  @Input()
+  public rundown: Rundown
+
+  public showStyleVariant: ShowStyleVariant
```

Comment:
Do we always have a value for this attribute?

Reply:
Not if we don't receive a valid response from the backend. For that reason i'm changing it to optional and guarding it when we set the default values. Good catch.